README.rst

```diff
@@ -59,6 +85,17 @@ You can also create additional redis client and process fixtures if you'd need t
     redis_my_proc = factories.redis_proc(port=None, logsdir='/tmp')
     redis_my = factories.redisdb('redis_my_proc')
```

Comment:
Maybe I can instantiate my own redis client in here (say [aioredis](https://aioredis.readthedocs.io/en/v1.3.0/`)) but I don't get how.

Reply:
You can't and you shouldn't have to do that. That client is for managing test data fixture (ie prepare data for the test itself), manage the database state in between tests and check for data state as part of the asserts.

Your code ought to make own separate connection, so it can use aioredis by itself.

Reply:
Ok, I was wondering if it was that, so it makes it strange for the end user to have both `redisdb` and its own client (the test code is very confusing). If I understand correctly you could have a decorator instead of feeding redis client as parameter (is that what they call "fixture"?). The test would have a function with an unused argument `redisdb`, but is still required for the test function to work properly?